src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -123,12 +123,19 @@ parameters:
     description: The identifier of a target node.
     required: true
     type: string
-  nodeFolderIdParam:
-    name: nodeId
-    in: path
-    description: The identifier of a folder node.
-    required: true
-    type: string
+  maxItemsForSizeParam:
+    name: maxItems
+    in: query
+    description: |
+      This assists in handling scenarios where ***large files*** or ***subfolders*** exist within a folder,
+      allowing the folder size determination mechanism to function in chunks.
+      
+      So that calculating the huge folder will be done efficiently.  
```

Comment:
This is a fragment of a sentence - could you expand it into a full sentence so that it's clear what you're referring to?

Reply:
We've refined the preceding message in single sentence.